Packages/com.chisel.editor/Chisel/Editor/Editor/Scene/Snapping/SnappingKeyboard.cs

```diff
@@ -17,14 +17,14 @@ private static void OnKeyboardEventCalled(KeyEventType type)
 		{
 			switch (type)
 			{
-				case KeyEventType.HalfGridSizeKey:			CSGSceneBottomGUI.DecreaseMoveSnapAll(); break;
-				case KeyEventType.DoubleGridSizeKey:		CSGSceneBottomGUI.IncreaseMoveSnapAll(); break;	
+				case KeyEventType.HalfGridSizeKey:			CSGSceneBottomGUI.ModifySnapDistance(0.5f); break;
+				case KeyEventType.DoubleGridSizeKey:		CSGSceneBottomGUI.ModifySnapDistance(2.0f); break;	
```

Comment:
Maybe have two floating point constants kHalfDistance = 0.5f and kDoubleDistance = 2.0f?

Reply:
Why?

Reply:
readability? Anyway, this is just a minor suggestion .. not really important

Reply:
I think after renaming the function to Multiply this should be much more clear, having extra variables seems like (minor) unnecessary overhead.

Reply:
Yeah that's fine, I don't feel strongly about this at all